packages/vue/src/utilities/directives/focus/focus-directive.js

```diff
@@ -3,9 +3,9 @@ export const focus = {
     el._mouseHandler = function () {
       el.style.outline = "none";
     };
-    el._keyHandler = function () {
+    el._keyHandler = function (e) {
```

Comment:
I tested this solution and it works ;) Well done! 
However I am not sure that this is the correct place to add this logic. You are modifying global `v-focus` directive to fix a problem that occurs in one case. Such a fix could cause issues in the future. In my opinion, the better solution would be to develop a local `v-focus` directive inside `SfInput` component with the logic you added here. @justyna-13 @AdamPawlinski  what do you think?